includes/admin/views/setup-wizard/main.php

```diff
@@ -27,9 +27,9 @@
 		</h1>
 
 		<ul class="llms-setup-progress">
-			<?php foreach ( $steps as $slug => $name ) : ?>
```

Comment:
we need to update all the view's `@version [version]` tags, and if you think the change is relevant also add a `@since [version] Summary of the change.`.